samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -2911,11 +2911,11 @@ private String commaDelimitedCalcQuestionAnswers(ItemDataIfc item, DeliveryBean
 		  // We need the key formatted in scientificNotation
 		  answer = service.toScientificNotation(answer, decimalPlaces);
 		  
-		  keysString = keysString.concat(answer + ",");
+		  keysString = keysString.concat(answer + ", ");
 		  answerSequence++;
 	  }
 	  if (keysString.length() > 2) {
```

Comment:
Should be this > 3?

Reply:
That was a bug, for a single answer with an integer of a single digit a comma was been shown. Currently in nightly looks this way:
![image](https://user-images.githubusercontent.com/19774177/159547407-498f58de-8f11-45ef-8e15-f905e5c5889f.png)